pkgs/by-name/pl/plow/package.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, plow }:
+
+buildGoModule rec {
+  pname = "plow";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "six-ddc";
+    repo = "plow";
+    rev = "c48d821994dbf216e35ea0108232840801952f13";
```

```suggestion
    rev = "refs/tags/v${version}";
```
The commit doesn't correspond to the tag.

---

You have to change `hash` if you change `rev`.

---

Set `hash = lib.fakeHash`, try to build and it will tell you the correct hash.

---

Ok it was the right one 👍